django/contrib/postgres/fields/ranges.py

```diff
@@ -159,6 +159,14 @@ def db_type(self, connection):
 RangeField.register_lookup(lookups.DataContains)
 RangeField.register_lookup(lookups.ContainedBy)
 RangeField.register_lookup(lookups.Overlap)
+DateRangeField.register_lookup(lookups.LowerInclusive)
+DateRangeField.register_lookup(lookups.LowerInfinite)
+DateRangeField.register_lookup(lookups.UpperInclusive)
+DateRangeField.register_lookup(lookups.UpperInclusive)
+DateTimeRangeField.register_lookup(lookups.LowerInclusive)
+DateTimeRangeField.register_lookup(lookups.LowerInfinite)
+DateTimeRangeField.register_lookup(lookups.UpperInclusive)
+DateTimeRangeField.register_lookup(lookups.UpperInclusive)
```

Comment:
These lookups should be available for all `RangeField`'s not only for `DateRangeField` and `DateTimeRangeField`.